src/components/NcKbd/NcKbd.vue

```diff
@@ -0,0 +1,244 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import type { Slot } from 'vue'
+
+import { computed } from 'vue'
+import { t } from '../../l10n.ts'
+import { isMac } from '../../utils/platform.ts'
+
+const {
+	symbol = undefined,
+	mac = isMac,
+} = defineProps<{
+	/**
+	 * Key name or symbol to display.
+	 * For common special keys (CTRL, ALT, SHIFT, Arrow keys, etc.) it will display the symbol on macOS and the localized name on Windows/Linux.
+	 */
+	symbol?: 'ArrowUp' | 'ArrowDown' | 'ArrowLeft' | 'ArrowRight' | 'Control' | 'Alt' | 'Shift' | 'Enter' | 'Space' | 'Tab' | 'Delete' | 'Escape' | (string & {})
```

Comment:
Just in case before merging.

@susnux Are you fine with this prop name?
(`key` cannot be used as it is a special attribute).
Alternative: `keyname`, `keycode` (but it has a different meaning in web api), `code`.

Reply:
As you also write "Key name" as the doc I think `keyname` is pretty good.
But also `symbol` seems to fit here so you can merged if you like ;)

Reply:
@Antreesy Do I remember it correctly, that in `useHotKey` the preferred way to set a key is the keyCode (like `KeyF`, not `F`)?

Reply:
No it takes `F` and then checks if `key` is equal (e.g. `F === F`) otherwise falls back to get the `keyCode` and removes the `Key` or `Digit` and then compares